packages/stores/src/providers/ExchangeRatesProvider.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react';
+import { useSubscription } from '@deriv/api';
+import useStore from '../useStore';
+
+const ExchangeRatesProvider = ({ children }: React.PropsWithChildren<unknown>) => {
+    const { data, subscribe } = useSubscription('exchange_rates');
+    const {
+        exchange_rates: { update },
+    } = useStore();
+
+    useEffect(() => {
+        subscribe({ payload: { base_currency: 'USD' } });
+    }, []);
+
+    useEffect(() => {
+        update(data);
+    }, [update, data]);
+
+    return <>{children}</>;
```

Comment:
Just for code standardisation.
```suggestion
    return <React.Fragment>{children}</React.Fragment>;
```